src/forms/DataSetElementCategoryComboSelectionDialog.component.js

```diff
@@ -31,7 +31,7 @@ const enhance = compose(
 );
 
 const getCategoryOptions = (categoryCombo) => {
-    if (categoryCombo.isDefault) {
+    if (!categoryCombo || categoryCombo.isDefault) {
         return "";
     } else {
         return !categoryCombo ? null : collectionToArray(categoryCombo.categories)
```

Comment:
This will always be `collectionToArray(categoryCombo.categories)` right?

Reply:
Indeed, now the second branch should be simplified. Will do on next task.